__init__.py

```diff
@@ -38,7 +38,6 @@ class VolumeSkill(MycroftSkill):
     MIN_LEVEL = 0
     MAX_LEVEL = 10
 
-    # TODO: Translation layer (have to match word in Level.voc)
```

Comment:
We probably want to keep a comment down in the new code to let people know that any words in `volume.words.value` must also be contained in `Level.voc`. At least, I'm presuming that is still the case.